.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you not add the .vs files or binary files (.suo, .db, .ipch, .sqlite) in the future?